IOSExercises/Views/ColorsTableViewController.m

```diff
@@ -0,0 +1,64 @@
+//
+//  ColorsTableViewController.m
+//  IOSExercises
+//
+//  Created by Arnold Joseph Caesar Esteban on 11/27/20.
+//  Copyright © 2020 Arnold Joseph Caesar Esteban. All rights reserved.
+//
+
+#import "ColorsTableViewController.h"
+#import "EXRColor.h"
+
+static NSString *kCellIdentifier = @"colorCell";
+
+@implementation ColorsTableViewController {
+    NSArray<EXRColor *> *_availableColorOptions;
```

Comment:
Avoid creating your own instance variables. Instead, create properties inside the private category:
```
@interface ColorsTableViewController () 

@property (nonatomic, strong) NSArray<EXRColor *> *availableColorOptions;

@end

@implementation ColorsTableViewController
...
```